module/userProjects/helper.js

```diff
@@ -473,11 +475,16 @@ module.exports = class UserProjectsHelper {
 						status: HTTP_STATUS_CODE.bad_request.status,
 					}
 				}
+				let kafkaUserProject = {
+					userId: userId,
+					projects: projectUpdated,
+				}
 				//  push project details to kafka
 				const kafkaPushedProject = await kafkaProducersHelper.pushProjectToKafka(projectUpdated)
+				const kafkaPushedUserProjects = await kafkaProducersHelper.pushUserActivitiesToKafka(kafkaUserProject)
 
 				console.log('<---------data sending to kafka -----------> : ', kafkaPushedProject)
```

Comment:
@praveenKDass ca we remove unused consoles

Reply:
Removed @VISHNUDAS-tunerlabs